docs/flutter/v2/guides/quickstart.mdx

```diff
@@ -5,316 +5,868 @@ nav: 2.0
+<Tab id="sample-app-0">
 
-#### iOS Permissions
+Please follow the below instructions to test the app for the android target platform:
 
-Add following permissions in iOS Info.plist file
+1. Allow camera, recording audio and internet permissions by adding the below snippet to the `AndroidManifest.xml` file (at the application tag level).
 
-```xml section=iOSPermissions sectionIndex=1
-<key>NSMicrophoneUsageDescription</key>
-<string>{YourAppName} wants to use your microphone</string>
+    <br />
 
-<key>NSCameraUsageDescription</key>
-<string>{YourAppName} wants to use your camera</string>
+    ```xml
+    <uses-feature android:name="android.hardware.camera" />
+    <uses-feature android:name="android.hardware.camera.autofocus" />
+    <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.CHANGE_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
+    <uses-permission android:name="android.permission.RECORD_AUDIO" />
+    <uses-permission android:name="android.permission.BLUETOOTH" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    ```
 
-<key>NSLocalNetworkUsageDescription</key>
-<string>{YourAppName} App wants to use your local network</string>
-```
+2. Add minimum SDK version (`minSdkVersion 21`) in "android/app/build.gradle" file (inside "defaultConfig").
 
-### Join a Video Call
+You will also need to request camera and record audio permissions at runtime before you join a call or display a preview. Please follow the [Android Documentation](https://developer.android.com/training/permissions/requesting#request-permission) for runtime permissions.
 
-To join a video call, call join method on HMSSDK with the config settings. To listen the updates, call `addUpdateListener` on HMSSDK.
+#### Build and run the app
 
-These are some listener's callback functions:
+-   Once you've made the above changes, your app is ready for testing. You can build the app and run it in an emulator or an actual android device.
+-   Go to _Run > Start debugging > select a device to use_ (android emulator or android phone connected via USB).
 
-`onJoin` - called when the join was successful and you have entered the room.
+Now, after you click `join`, you should be able to see yourself (android emulator doesn't support actual video, you can connect an actual device to see your video in real-time). You can join the room using a browser as the second peer to check audio/video transactions between two or more peers.
 
-> 💡 Audio will be automatically connected, video requires some work on your side.
+<video loop="true" autoplay="autoplay" controls="controls" id="vid" muted>
```

Comment:
It would be better if the video showed running on Android/iPhone devices instead of Emulators. 
Since emulators do not publish video, first-time users get the impression that 100ms is not working.